lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -2,22 +2,21 @@ const ChangeTypes = require('../../ChangeTypes');
 const Node = require('../../node');
 const TreeSearch = require('../../TreeSearch');
 
-// If `node` is a product of terms where some are fractions (but none are
-// polynomial terms), multiplies them together.
+// If `node` is a product of terms where:
+// 1) at least one is a fraction
+// 2) either none are polynomial terms, OR
+//    at least one has a symbol in the denominator
+// then multiply them together.
```

Comment:
this docstring is 👌 

is it noted somewhere why we have the case of none are polynomial terms?